### DISCUSSION TITLE
Getting a 403 forbidden for the ruleset from my Render service

### DISCUSSION THREAD

**Discussion by zachlandes** [kubero-dev/ladder#85] [Q&A] [answered]
I am attempting to host this app from Render.com, but setting an environment variable of RULESET with value = https://t.ly/14PSf, I get a 403 when my service starts up. Are there any workarounds? Have I set this up incorrectly? I am using the docker image. 

Here's the startup message in Render:

```
2024/10/01 00:01:18 WARN: failed to load ruleset from 'https://t.ly/14PSf'
failed to load rules from remote url (403 Forbidden) on 'https://t.ly/14PSf'
WARN: failed to load 1 rulesets
```

**Comment by mms-gianni** [comment, marked as answer]
have you tried to use : https://raw.githubusercontent.com/everywall/ladder-rules/main/ruleset.yaml

**Comment by zachlandes** [comment]
Ah that's a good solution, thank you!
